infrastructure/lambda-update-doc-ref.tf

```diff
@@ -0,0 +1,93 @@
+module "update_doc_alarm" {
+  source               = "./modules/lambda_alarms"
+  lambda_function_name = module.update-doc-ref-lambda.function_name
+  lambda_timeout       = module.update-doc-ref-lambda.timeout
+  lambda_name          = "update_document_reference_handler"
+  namespace            = "AWS/Lambda"
+  alarm_actions        = [module.update_doc_alarm_topic.arn]
+  ok_actions           = [module.update_doc_alarm_topic.arn]
+  depends_on           = [module.update-doc-ref-lambda, module.update_doc_alarm_topic]
+}
+
+
+module "update_doc_alarm_topic" {
```

Comment:
name should be consistent. lambda is update-doc-ref-lambda, this should be update-doc-ref-alarm-topic